spec/services/recurring_contribution/subscriptions/update_status_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe RecurringContribution::Subscriptions::UpdateStatus do
+  include RecurringContributionsHelper
+
+  describe ".process" do
+    let(:pagarme_subscription) { build_subscription_mock('credit_card') }
+    let!(:juntos_subscription) { create(:subscription, subscription_code: pagarme_subscription.id, status: 'pending_payment') }
+    let(:update_subscription_service) { described_class.process(pagarme_subscription.id, current_status) }
+
+    context "when the sthe current_status passed as param is one of the permitted values" do
+      let(:current_status) { 'paid' }
+
+      it "should return a Subscription with an updated status" do
```

Comment:
I would avoid using should at the description as well, just inflecting the verb like this:
```ruby
it "returns ...."
```

I would also change this description to:

```ruby
it "updates Subscription status" do
```